### ISSUE TITLE
[brightcove/smithsonian] Better support requested

### ISSUE THREAD

**Issue by keybounce** [ytdl-org/youtube-dl#12842]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like that [x])
- Use *Preview* tab to see how your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2017.04.26*. If it's not read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x ] I've **verified** and **I assure** that I'm running youtube-dl **2017.04.26**

### Before submitting an *issue* make sure you have:
- [ x] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [ ]x [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [ ] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [x ] Other

---

I am looking for better support for Brightcove/Legacy / Smithsonian Channel.

Here is what I get when trying to find series/episode information:

```keybounceMBP:Profiles michael$ youtube-dl -v --get-filename http://www.smithsonianchannel.com/videos/oracles/55641
[debug] System config: []
[debug] User config: [u'-o', u'%(series)s/s%(season_number)02d-e%(episode_number)02d-%(title)s.%(ext)s', u'-f', u'\nbest[ext=mp4][height>431][height<=576]/\nbestvideo[ext=mp4][height=480]+bestaudio[ext=m4a]/\nbest[ext=mp4][height>340][height<=431]/\nbestvideo[ext=mp4][height>360][height<=576]+bestaudio/\nbest[height>340][height<=576]/\nbestvideo[height>360][height<=576]+bestaudio/\nbestvideo[height=360]+bestaudio/\nbest[ext=mp4][height>=280][height<=360]/\nbest[height<=576]/\nworst', u'--ap-mso', u'Dish', u'--ap-username', u'PRIVATE', u'--ap-password', u'PRIVATE', u'--write-sub', u'--write-auto-sub', u'--sub-lang', u'en,enUS,en-us', u'--sub-format', u'ass/srt/best', u'--convert-subs', u'ass', u'--embed-subs', u'--recode-video', u'mp4', u'--mark-watched']
[debug] Custom config: []
[debug] Command-line args: [u'-v', u'--get-filename', u'http://www.smithsonianchannel.com/videos/oracles/55641']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2017.04.26
[debug] Python version 2.7.5 - Darwin-13.4.0-x86_64-i386-64bit
[debug] exe versions: ffmpeg 3.2.4, ffprobe 3.2.4, rtmpdump 2.4
[debug] Proxy map: {}
WARNING: Falling back on generic information extractor.
NA/sNA-eNA-Sacred Sites - Oracles.mp4
keybounceMBP:Profiles michael$ 
```
If I try to download, I get no subtitles/closed captions: (NB: done with a different -o, and actually before that verbose one)
```
keybounceMBP:Oracles michael$ youtube-dl http://www.smithsonianchannel.com/videos/oracles/55641
[generic] 55641: Requesting header
WARNING: Falling back on generic information extractor.
[generic] 55641: Downloading webpage
[generic] 55641: Extracting information
[download] Downloading playlist: Sacred Sites: Oracles
[generic] playlist Sacred Sites: Oracles: Collected 1 video ids (downloading 1 of them)
[download] Downloading video 1 of 1
[brightcove:legacy] 5385068692001: Downloading webpage
[brightcove:legacy] 5385068692001: Extracting information
[hlsnative] Downloading m3u8 manifest
[hlsnative] Total fragments: 306
[download] Destination: sNAeNA - Sacred Sites - Oracles.mp4
[download] 100% of 685.11MiB in 05:34
[ffmpeg] Fixing malformated aac bitstream in "sNAeNA - Sacred Sites - Oracles.mp4"
[ffmpeg] Not converting video file sNAeNA - Sacred Sites - Oracles.mp4 - already is in target format mp4
[ffmpeg] There aren't any subtitles to convert
[ffmpeg] There aren't any subtitles to embed
[download] Finished downloading playlist: Sacred Sites: Oracles
keybounceMBP:Oracles michael$ 
```

(it was the lack of subtitles that made me check for more details)

**Comment by dstftw** [comment]
There is no series metadata available as well as subtitles. Moreover it does not even play in browser for me.

**Comment by remitamine** [comment]
subtitles available using `BrightcoveNewIE`.
```
youtube-dl --list-subs 'https://secure.brightcove.com/services/viewer/federated_f9/?isVid=1&isUI=1&playerID=1466806621001&autoStart=true&videoId=5385068692001&secureConnections=true&secureHTMLConnections=true'
[brightcove:legacy] 5385068692001: Downloading webpage
[brightcove:legacy] 5385068692001: Extracting information
5385068692001 has no subtitles
```
```
youtube-dl --geo-bypass-country US --list-subs https://players.brightcove.net/376817008/306aa4cf-c502-462b-933b-fdb32bce1826_default/index.html?videoId=5385068692001
[brightcove:new] 5385068692001: Downloading webpage
[brightcove:new] 5385068692001: Downloading JSON metadata
[brightcove:new] 5385068692001: Downloading m3u8 information
[brightcove:new] 5385068692001: Downloading m3u8 information
Available subtitles for 5385068692001:
Language formats
en       vtt
```

**Comment by yan12125** [comment]
This website uses a custom brightcove:new player (data-bcid handled in application.js and then passed to brightcove's index.js)
